src/Raven.Studio/languageService/test/grammar/orderBy.spec.ts

```diff
@@ -0,0 +1,26 @@
+﻿import {parseRql} from "../../src/parser";
+import {
+    OrderByItemContext,
+    OrderByStatementContext
+} from "../../src/generated/RqlParser";
+
+describe("ORDER BY statement parser", function () {
+    it("single", function () {
+        const {parseTree, parser} = parseRql("from test order by item as ALPHANUMERIC desc");
+
+        expect(parser.numberOfSyntaxErrors)
+            .toEqual(0);
+
+        const orderBy = parseTree.orderByStatement();
+        
+        expect(orderBy)
+            .toBeInstanceOf(OrderByStatementContext);
+        expect(orderBy._value)
+            .toBeInstanceOf(OrderByItemContext);
+        
+        const item = orderBy._value;
+        expect(item._value.text).toEqual("item")
+        expect(item._order.text).toEqual("asALPHANUMERIC");
```

Comment:
avoid such text to compare

Reply:
?
Changed into:
![image](https://user-images.githubusercontent.com/86351904/133973188-8ad205aa-0ae2-4850-8437-8bdec729dfe0.png)